respy/pre_processing/specification_helpers.py

```diff
@@ -121,4 +128,41 @@ def _base_row(index_tuple, data):
     ind = pd.MultiIndex.from_tuples([index_tuple], names=["category", "name"])
     cols = ["value", "comment"]
     df = pd.DataFrame(index=ind, columns=cols, data=[data])
+
     return df
+
+
+def observable_prob_template(observables):
+    to_concat = []
+    for i in range(len(observables)):
+        probs = np.random.uniform(size=observables[i])
+        probs /= probs.sum()
+
+        for j in range(observables[i]):
+            ind = (f"observables", f"observable_{i}_{j}")
+            dat = [probs[j], f"Probability of observable {i} being level choice {j}"]
+            to_concat.append(_base_row(ind, dat))
+
+    return pd.concat(to_concat, axis=0, sort=False)
+
+
+def observable_coeffs_template(observables, template):
+    index = {
+        x for x in template.index.get_level_values(0) if "nonpec" in x or "wage" in x
+    }
+
+    labels = generate_obs_labels(observables, index)
+    to_concat = []
+    for y in labels:
+        dat = [0, f"effect of {y[1]}"]
+        to_concat.append(_base_row(y, dat))
+    return pd.concat(to_concat, axis=0, sort=False)
```

Comment:
I like to have a blank line before return statements to end logical blocks.

```suggestion

    return pd.concat(to_concat, axis=0, sort=False)
```